hardware_interface/include/hardware_interface/controller_info.hpp

```diff
@@ -32,6 +33,9 @@ struct ControllerInfo
   /// Controller type.
   std::string type;
 
+  /// Controller param file
+  std::optional<std::string> params_file;
```

Comment:
I would keep the long name :)
```suggestion
  std::optional<std::string> parameterss_file;
```

Reply:
But only very personal oppinion.

Reply:
Sure. I can make the change. 
As I need to change in multiple places, let me do it offline and push the changes

Reply:
Renamed as you suggested!